src/auth/auth.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
```

Comment:
Is there a reason the auth module is not under `/modules/auth`?

Reply:
in reality, it can be anywhere
We tend to have at the top level items that are globally used - i.e. `/config`, `/database`, /helpers` etc
Although this can easily be as part of the `modules/` folder, so happy to move it if needed

Reply:
The reason I ask is that I think it's a bit surprising to have a folder called `modules` but it only has some of the `modules` in it rather than all of them. I have a preference for moving them to the `modules` folder (or renaming the `modules` folder) but if the rest of the team prefer this then that's fine with me :)

Reply:
Folder `modules` contain mdm.module.ts , so `modules` could be renamed to `mdm` or `mdm-endpoints`

Reply:
this is updated now